src/main/java/net/william278/profanitycheckerapi/ProfanityChecker.java

```diff
@@ -66,6 +70,90 @@ public double getTextProfanityLikelihood(String text) {
         return interpreter.getValue("predict_prob([text])[0].item()", Double.class);
     }
 
+    /**
+     * Determines whether a string of text contains profanity
+     *
+     * @param text The string of text to check
+     * @return {@code true} if the text likely contains profanity; {@code false} otherwise
+     */
+    public boolean isTextProfaneBypass(String text) {
+        Set<String> toCheck = getTextToCheck(text);
+        for (String s : toCheck) {
+            interpreter.set("text", s);
+            boolean result = interpreter.getValue("predict([text])[0].item()", Integer.class) == 1;
+            if (result) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Returns a {@code double} value (0 to 1 inclusive) indicating the probability that the machine learning algorithm thinks the string contains profanity
+     *
+     * @param text The string of text to check
+     * @return A {@code double} ranging between 0 and 1 inclusive that indicates the likelihood the string of text contains profanity
+     */
+    public double getTextProfanityLikelihoodBypass(String text) {
+        Set<String> toCheck = getTextToCheck(text);
+        double highest = 0;
+        for (String s : toCheck) {
+            interpreter.set("text", s);
+            double result = interpreter.getValue("predict_prob([text])[0].item()", Double.class);
+            if (result > highest) {
+                highest = result;
+            }
+        }
+        return highest;
+    }
+
+    /**
+     * Determines whether a string of text contains profanity
+     *
+     * @param text The string of text to check timeLimitMilli
+     * @param timeLimitMilli The time limit in milliseconds
+     * @return {@code true} if the text likely contains profanity; {@code false} otherwise
+     */
+    public boolean isTextProfaneBypassTimed(String text, long timeLimitMilli) {
```

Comment:
I think we can reduce code repetition with this method, too, by making the above two methods call this method and just passing `Long.MAX_VALUE` as the time limit.